addons/l10n_au/data/account_tax_report_data.xml

```diff
@@ -80,14 +81,15 @@
         <field name="name">G9: GST on sales (G8 divided by eleven)</field>
         <field name="sequence" eval="10"/>
         <field name="formula">((G1-(G2+G3+G4))+G7)/11</field>
-        <field name="parent_id" ref="account_tax_report_gstrpt_sale_total"/>
+        <field name="parent_id" eval="False"/>
```

Comment:
I'm not sure why you want to change this if you are using `formula=None` for `account_tax_report_gstrpt_sale_total` anyways.

This will just remove this line from the report, is that what you want?

Reply:
@william-andre 

Changing this leaves it on the report, but in a bold attribute, not as a sub attribute.  This increases the consistency in the report between sales and purchases section.

With this change, the visual layout more closely resembles the layout from the Australian Taxation Office.

We use this mod already, directly with our clients.

Reply:
> Changing this leaves it on the report, but in a bold attribute, not as a sub attribute. 

My bad, I was confused because that behavior has been changed in newer versions.

________________
> This increases the consistency in the report between sales and purchases section.

What do you mean?
https://github.com/odoo/odoo/blob/d6ff8c49d7a0775c4744dfecdeb679595b791e76/addons/l10n_au/data/account_tax_report_data.xml#L182-L188

> With this change, the visual layout more closely resembles the layout from the Australian Taxation Office.

I couldn't find what you are referring to, I only found this
![image](https://aws1.discourse-cdn.com/business5/uploads/manager1/original/3X/d/9/d9dd065de035e23b31edb0f7f07b58347330eb22.png)
![image](https://docplayer.net/docs-images/50/15476020/images/page_7.jpg)
And what seems to be the official document: https://www.ato.gov.au/assets/0/104/694/815/68f311eb-9d6a-4367-8ea5-3dbb52fb859e.pdf

Reply:
@william-andre Essentially, the 3 figures that are important are G9, which gets transferred to 1A, and G20, which gets transferred to 1B.

In the existing report, G20 is printed on the Odoo report without a parent id, and stands out nice and cleanly, but G9 is lost in the top section.

Essentially, the report makes it look like a subtotal, but it is not - it is another figure / 11 - to be transferred,  When we were queried about it, we checked with a few of our customers and the (small) sample we took all liked the visual improvement and consistency of changing G9 to parallel G20.

Reply:
@william-andre Ahh, wait a sec, I see with the parent Id - I need to check my local code - hang on...

Reply:
@william-andre Sorry, was a while since I looked at this.  Check account_tax_report_gstrpt_g20b, which is the sum of g20a and gst_only_purchases.... That is what is printing as "G20"

Reply:
![Screenshot from 2022-01-13 18-39-45](https://user-images.githubusercontent.com/1011100/149381343-710494f9-1a74-4acb-95cb-00f090cf7724.png)

Reply:
Looking at what seems to be the official document: https://www.ato.gov.au/assets/0/104/694/815/68f311eb-9d6a-4367-8ea5-3dbb52fb859e.pdf, it looks like that it the mistake, and not the opposite.

Reply:
I see what you are saying - but in the end, the 2 figures that have to be extracted are G9 and G20 - and they do not stand out in the current format.

IF we were to introduce, as hinted, a "Transfer to 1A / 1B" or "Summary" as hinted by the main screenshots, I would agree.
![Screenshot from 2022-01-14 09-38-50](https://user-images.githubusercontent.com/1011100/149482223-31258967-bafc-4307-8453-9ede353f9f19.png)

![Screenshot from 2022-01-14 09-38-38](https://user-images.githubusercontent.com/1011100/149482218-7955644c-2165-4088-a398-fc1afdab024d.png)

Reply:
Adding these 1A and 1B make more sense to me than hacking the hierarchy, and keeps things more standard.
Every single other software I saw does it like that.